src-overlay-ui/styles/_animations.scss

```
@@ -0,0 +1,158 @@
+@use 'tokens' as *;
+
+// Classes for `animate.enter` / `animate.leave`. Every one reads `--anim-duration` and
+// `--anim-delay`, and the fly ones `--anim-fly-y`, so a call site can retune a single usage:
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)